docs/shared/mpas_init.yaml

```diff
@@ -1,46 +1,44 @@
 user:
```

Comment:
Just making this file a bit more consistent with what I was trying for in the docs.